### DISCUSSION TITLE
git storage: have files from default language in locale directory rather than root

### DISCUSSION THREAD

**Discussion by ashledombos** [requarks/wiki#2831] [Help / Questions]
I have several languages, base is English, other are translations of the base documents.
I use git as storage, and Transifex as translation system.
The issue I face is default locale documents are at the root, 

DirA / DocA.md
DocB.md
DocC.md

the translations are in subdirectories named by locale, like

it/DirA/DocA.md
it/DocB.md
it/DocC.md

If I import the root directory in transifex as source, all the locale subdirectories are considered as source document also.

Is there a way to completely separate the default language in its own locale directory (in my case /en/) to avoid this?

**Comment by lipanpan-hub** [comment]
anyone resolve ?
